.github/workflows/create-pr.yml

```diff
@@ -11,11 +11,19 @@ jobs:
 
     runs-on: ubuntu-latest
     steps:
+      # Get GitHub token via the CT Changesets App
+      - name: Generate GitHub token (via CT Changesets App)
+        id: generate_github_token
+        uses: tibdex/github-app-token@v2.1.0
+        with:
+          app_id: ${{ secrets.CT_CHANGESETS_APP_ID }}
+          private_key: ${{ secrets.CT_CHANGESETS_APP_PEM }}
+          
       - name: Checkout
         uses: actions/checkout@v3
       - uses: jenschude/auto-create-pr-action@v0.3.1
         with:
           request_title: "Update generated SDKs"
-          github_token: ${{ secrets.GITHUB_TOKEN }}
+          github_token: ${{ steps.generate_github_token.outputs.token }}
```

Comment:
Do you know what token is this? It's called `GITHUB_TOKEN` and not `RELEASE_GITHUB_TOKEN` 🤔

Reply:
It doesn't exist on the repo as a secret. It's the "default" token. Maybe we don't have to remove it then 🤔 

https://docs.github.com/en/actions/security-guides/automatic-token-authentication

WDYT?

Reply:
Ah ok, then I would leave it as-is

Reply:
Done